apps/agreement/contracts/Agreement.sol

```diff
@@ -270,12 +291,17 @@ contract Agreement is IAgreement, AragonApp {
         // An initial collateral requirement is created when disputable apps are activated, thus length will be always greater than 0
         uint256 currentCollateralRequirementId = disputableInfo.nextCollateralRequirementsId - 1;
         CollateralRequirement storage requirement = _getCollateralRequirement(disputableInfo, currentCollateralRequirementId);
-        _lockBalance(requirement.staking, _submitter, requirement.actionAmount);
-        // TODO: pay court transaction fees
+        Staking staking = requirement.staking;
+        _lockBalance(staking, _submitter, requirement.actionAmount);
+
+        // Pay court transaction fees
+        IDisputable disputable = IDisputable(msg.sender);
 
         uint256 id = nextActionId++;
+        _payAppFees(currentSettingId, disputable, staking, _submitter, id);
```

Comment:
Just wondering if we shouldn't do this last?

We may also want to do this after emitting the `ActionSubmitted` event—if someone's looking through the transaction's logs, it may more sense if they see the `ActionSubmitted` even before any fee paid events.